controller/users.go

```diff
@@ -133,11 +134,16 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 		// to have everything - whatever we are updating, and whatever are not.
 		keycloakUserProfile := copyExistingKeycloakUserProfileInfo(keycloakUserExistingInfo)
 
-		// Disabling updation of email till we figure out how to do the same in Keycloak Error-free.
-		//
-
 		updatedEmail := ctx.Payload.Data.Attributes.Email
 		if updatedEmail != nil {
+			isUnique, err := isEmailUnique(appl, *updatedEmail, *user)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errors.Wrap(err, fmt.Sprintf("error updating user with id %s", identity.UserID.UUID)))
```

Comment:
We widely use identity ID instead of UserID. So reporting error for the identity ID would be more informative. You can mention both identity and user IDs though.